### ISSUE TITLE
[image-builder] create baseImage and workspaceImage repos if they do not exist (AWS only)

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/gitpod#15291]
## Is your feature request related to a problem? Please describe

Automatically create the `baseImageRepository` and `workspaceImageRepository` if using ECR, when the repositories do not exist already in ECR.

Before starting this issue, [please confirm which IAM permissions are needed](https://github.com/gitpod-io/gitpod/issues/14914#issuecomment-1345827911).

## Describe the behaviour you'd like

Have `bob proxy` create the missing ECR repos, before it pushes images to them.

## Describe alternatives you've considered

Alternatively, `image-builder-mk3` may be able to do this on build.

## Additional context

We learned [here](https://github.com/gitpod-io/gitpod/issues/14914#issuecomment-1345751478) that the repos must be created manually.

**Comment by kylos101** [comment]
@utam0k @jenting this is no longer needed.

The Scout Team is working on as output [here](https://www.notion.so/gitpod/Dedicated-Stages-c1456a34821d4c43a3b8b3a4151d20e0#0c7bf6fe171e4b8091390ff70ee06e40). [[1](https://github.com/gitpod-io/scout-foundation/pull/183)][[2](https://github.com/gitpod-io/scout-foundation/pull/194)]
